jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -815,7 +815,7 @@ public void write(boolean last, ByteBuffer byteBuffer, Callback callback)
             {
                 if (BufferUtil.hasContent(byteBuffer))
                 {
-                    if (isStreaming())
+                    if (!isWriting())
```

Comment:
Just a niggle, but can we invert the if so the writing block becomes before the else.

Reply:
Done.

Reply:
@lorban not really dramatic here, but worth mentioning. When you make changes to an existing commit, please add a new commit with your changes rather than amend-committing, especially other people's commits. In this case it's a trivial change (revert the logical order of an if/else branch pair). But as you can imagine this situation could be difficult to handle in the general sense.

Reply:
I'll try to keep that in mind, thanks for  the tip.